docs/intro/agload.md

```diff
@@ -32,7 +32,7 @@ load_labels_from_file('<graph name>',
                       false)
 ```
 
-Function `load_edges_from_file` can be used to load properties from the CSV file. Please see the file structure in the following. 
+Function `load_edges_from_file` can be used to load edges from the CSV file. Please see the file structure in the following. 
```

Comment:
Changed this to conform with the prior sentence, which said "(...) to load ***vertices*** from the CSV files. "